ui/src/reducers/panelData.js

```diff
@@ -52,6 +54,18 @@ export function panelData(state = initialData, action) {
 
 		return newState;
 
+	case CLONE_PANEL:
+		const clonedPanel = _.cloneDeep(action.data.panels[0]);
+
+		clonedPanel.panels = [];
+		clonedPanel.data.title += ` (${UI_I18N['message.cloned_title']})`;
```

Comment:
I think the blank space character and the parentheses themselves should be part of the localized string since the order may be different in another language.